examples/terraform/network.tf

```diff
@@ -0,0 +1,36 @@
+#
+# VPC resources
+#
+data "aws_ami" "amazon_linux" {
+  most_recent = true
+  owners      = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["amzn2-ami-hvm*"]
+  }
+}
+
+module "vpc" {
+  source = "../../"
+
+  name        = join("", ["vpc", var.environment, var.project])
+  region      = var.region
+  bastion_ami = data.aws_ami.amazon_linux.id
+}
+
+resource "aws_ssm_document" "default" {
+  name          = "SSM-SessionManagerRunShell"
+  document_type = "Session"
+
+  content = jsonencode({
+    schemaVersion = "1.0"
+    description   = "Document to hold regional settings for Session Manager"
+    sessionType   = "Standard_Stream"
+    inputs = {
+      shellProfile = {
+        linux = "cd ~ && /usr/bin/env bash"
+      }
+    }
+  })
+}
```

Comment:
I don't think we'd need to re-write the IAM role, we already allow SSM connection to non-prd, just because that the `ssm-user` has sudo privileges by default, I'd rather disable it and use `ec2-user` with a keypair, then connect to the bastion with ssh and use ssm as a proxycommand. That way we only have one sudo user on that bastion, we can remove the `aws_ssm_document` resource, and the only change we'd need to make to this module to make that work is to add a variable for keypair and pass it to the bastion instance.